sw/device/tests/rv_core_ibex_address_translation_test.c

```diff
@@ -58,15 +58,19 @@ const dif_rv_core_ibex_addr_translation_slot_t kSlots[] = {
     kDifRvCoreIbexAddrTranslationSlot_0, kDifRvCoreIbexAddrTranslationSlot_1};
 
 // Translation descriptions to use.
+// Junk bits (0xDEADBEEF) are added to the `remap_addr` fields to ensure they
+// are ignored by the translation mechanism.
 static const dif_rv_core_ibex_addr_translation_mapping_t kMakeLowerCaseMapping =
     {
         .matching_addr = (uintptr_t)remapped_function,
-        .remap_addr = (uintptr_t)make_lower_case,
+        .remap_addr = (uintptr_t)make_lower_case +
+                      (uintptr_t)(0xDEADBEEF & (kRemapAlignment - 1)),
         .size = kRemapAlignment,
 };
 static const dif_rv_core_ibex_addr_translation_mapping_t kGetNameMapping = {
     .matching_addr = (uintptr_t)remapped_function,
-    .remap_addr = (uintptr_t)get_name,
+    .remap_addr =
+        (uintptr_t)get_name + (uintptr_t)(0xDEADBEEF & (kRemapAlignment - 1)),
```

Comment:
Wouldn't it be better to randomize the lower bits instead of making them constant ones?

Reply:
Maybe though in general the chip-level tests haven't been using randomization so it seems reasonable to not randomize here as well.

Reply:
Ok, I'll leave this up to you. I've approved it in the meantime to unblock